mautic/mautic
# Reduce ajax call for email count stats

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
For big instances calculate count stats from 10 emails in ajax call take a long time and make really slow experience with work with Mautic. I think same noticed @alanhartless times ago, do you have something to add?

This is our time to load each ajax call before update.

![image](https://user-images.githubusercontent.com/462477/108820931-2da52b00-75bd-11eb-8bb4-a8f5ade64170.png)




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Check If email stats are loaded properly
